layouts/fastly/helix.vcl

```diff
@@ -1231,6 +1231,11 @@ sub hlx_deliver_errors {
 sub hlx_error_errors {
   set obj.http.X-Trace = req.http.X-Trace + "; hlx_error_errors(" obj.status ")";
   # Cache Condition: OpenWhisk Error Prio: 10
+  if (obj.status == 404 ) {
```

Comment:
We need to be careful here so that we do not override a custom `404.html` if it has been provided by `helix-dispatch`.

Reply:
for now I am trying to understand why I do not get the default 404.html defined in the VCL.
how should I set up the custom error page ? `/htdocs/404.html` ?

Reply:
Either `htdocs/404.html` or `/404.html`.